static/app/views/performance/newTraceDetails/trace.tsx

```diff
@@ -322,10 +321,42 @@ function Trace({
       pathname: location.pathname,
       query: queryParamsWithoutNode,
     });
+
+    setDetailNode(null);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  useOnClickOutside(containerRef.current, onOutsideClick);
+  // Unhighlight the row when clicking outside of the trace or the detail panel
+  useEffect(() => {
+    const listener = (event: MouseEvent | TouchEvent) => {
+      if (!containerRef.current) {
+        return;
+      }
+
+      const traceViewElement: HTMLDivElement | null =
```

Comment:
```suggestion
      const traceElement: HTMLDivElement | null =
```

Reply:
Minor, but you should just use HTMLElement types here since the tagname doesnt really matter. If we were to ever change the container type, you wouldnt need to update the type signatures